host-configs/docker/docker-linux-ubuntu18.04-x86_64-clang@10.0.0.cmake

```diff
@@ -0,0 +1,75 @@
+#------------------------------------------------------------------------------
+# !!!! This is a generated file, edit at own risk !!!!
+#------------------------------------------------------------------------------
+# SYS_TYPE: linux-ubuntu18.04-ivybridge
+# Compiler Spec: clang@10.0.0
+#------------------------------------------------------------------------------
+# CMake executable path: /usr/bin/cmake
+#------------------------------------------------------------------------------
+
+#------------------------------------------------------------------------------
+# Compilers
+#------------------------------------------------------------------------------
+
+set(CMAKE_C_COMPILER "/usr/bin/clang" CACHE PATH "")
+
+set(CMAKE_CXX_COMPILER "/usr/bin/clang++" CACHE PATH "")
+
+set(ENABLE_FORTRAN ON CACHE BOOL "")
+
+set(CMAKE_Fortran_COMPILER "/usr/bin/gfortran" CACHE PATH "")
+
+set(BLT_EXE_LINKER_FLAGS " -Wl,-rpath,/usr/lib" CACHE PATH "Adds a missing libstdc++ rpath")
+
+#------------------------------------------------------------------------------
+# TPLs
+#------------------------------------------------------------------------------
+
+# Root directory for generated TPLs
+set(TPL_ROOT "/home/axom/axom_tpls/clang-10.0.0" CACHE PATH "")
+
+set(CONDUIT_DIR "${TPL_ROOT}/conduit-master" CACHE PATH "")
+
+# MFEM not built
```

Comment:
Since we're getting rid of the other clang docker images, we should probably enable MFEM in this one.
(This could be a future task though).

Reply:
i can rebuild it later after this is working